entries/c/coindcx.com.json

```
@@ -0,0 +1,15 @@
+{
+    "CoinDCX": {
+      "domain": "coindcx.com",
+      "tfa": [
+        "sms",
+        "email",
+        "totp"
+      ],
+      "documentation": "https://support.coindcx.com/hc/en-gb/articles/360003181855-How-to-secure-your-account-with-Two-factor-Authentication-2FA-",
```

The text after the numbers in the URL isn't needed.
```suggestion
      "documentation": "https://support.coindcx.com/hc/en-gb/articles/360003181855",
```